execution/engine/engine.go

```diff
@@ -130,11 +130,11 @@ func (ee *Engine) NotifyForkchoiceUpdate(
 	return payloadID, latestValidHash, nil
 }
 
-// VerifyAndNotifyNewPayload verifies the new payload and notifies the
+// NotifyNewPayload verifies the new payload and notifies the
```

Comment:
It does not verify anymore, sth like:

```suggestion
// NotifyNewPayload notifies the execution client of the new payload
```